fastlane/spec/helper/xcodes_helper_spec.rb

```diff
@@ -0,0 +1,66 @@
+describe Fastlane::Helper::XcodesHelper do
+  describe "#read_xcode_version_file" do
+    let(:xcode_version_path) { ".xcode-version" }
+
+    context ".xcode-version file exists" do
+      before do
+        allow(Dir).to receive(:glob).with(".xcode-version").and_return([xcode_version_path])
+        allow(File).to receive(:read).with(xcode_version_path).and_return("14")
+      end
+
+      it "return its content" do
+        expect(subject.class.read_xcode_version_file).to eq("14")
+      end
+    end
+
+    context "no .xcode-version file exists" do
+      before do
+        allow(Dir).to receive(:glob).with(".xcode-version").and_return([])
+      end
+
+      it "return nil" do
+        expect(subject.class.read_xcode_version_file).to eq(nil)
+      end
+    end
+  end
+
+  describe "#find_xcodes_binary_path" do
+    it "returns the binary path when it's installed" do
+      expect(subject.class).to receive(:find_xcodes_binary_path).and_return("/opt/homebrew/bin/xcodes")
+      expect(subject.class.find_xcodes_binary_path).to eq("/opt/homebrew/bin/xcodes")
+    end
+
+    it "returns a string that doesn't point to a file that exists, when it's not installed" do
+      expect(subject.class).to receive(:find_xcodes_binary_path).and_return("xcodes not found")
+      expect(File.exist?(subject.class.find_xcodes_binary_path)).to eq(false)
+    end
+  end
+
+  describe "Verify" do
+    describe "#requirement" do
+      context "when argument is missing or empty" do
+        let(:argument) { nil }
+
+        it "raise user error" do
+          expect { subject.class::Verify.requirement(nil) }.to raise_error(FastlaneCore::Interface::FastlaneError, /Version must be specified/)
```

Comment:
```suggestion
          expect { subject.class::Verify.requirement(nil) }.to raise_error(FastlaneCore::Interface::FastlaneError, 'Version must be specified')
```

? Or am I missing something? 🤔

Reply:
Nope, you're right :)

Reply:
```suggestion
          expect { subject.class::Verify.requirement(argument) }.to raise_error(FastlaneCore::Interface::FastlaneError, 'Version must be specified')
```